src/components/product-card/ProductCard.jsx

```diff
@@ -1,6 +1,19 @@
-const ProductCard = ({item}) => {
+import { useCart } from "../../context/cart-context";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+
+const ProductCard = ({ item }) => {
+  const { addToCartHandler, cartList } = useCart();
+  let navigate =useNavigate();
+  const goToCart = () => {
+    navigate("/cart");
+  };
   return (
-    <div style={item.includeStock===false ? {opacity:0.3} : {opacity: 1}} className="card-vertical box-shadow" key={item.id}>
+    <div
+      style={item.includeStock === false ? { opacity: 0.3 } : { opacity: 1 }}
```

Comment:
avoid inline styling

Reply:
I dont have approach how to do in css to select styling. i will do it later.